doc/theoretical_description_conformity_scores.rst

```diff
@@ -0,0 +1,113 @@
+
+Where :math:`q(s)` is the :math:`(1-\alpha)` quantile of the conformity scores.
+(see :doc:`theoretical_description_regression` for more details).
+
+With this score the intervals of predictions will be constant over the whole dataset.
+This score is by default symmetric (*see above for definition*).
+
+2. The gamma score
+==================
+
+The gamma score (:class:`mapie.conformity_scores.GammaConformityScore`) adds a
+notion of adaptivity with the normalization of the residuals by the predictions.
+
+.. math:: \frac{|Y-\hat{\mu}(X)|}{\hat{\mu}(X)}
+
+It computes adaptive intervals : intervals of different size on each example, with
+the following formula  :
+
+.. math:: [\hat{\mu}(X) * (1 - q(s)), \hat{\mu}(X) * (1 + q(s))]
+
+Where :math:`q(s)` is the :math:`(1-\alpha)` quantile of the conformity scores.
+(see :doc:`theoretical_description_regression` for more details).
+
+This score is by default asymmetric (*see definition above*).
+
+Compared to the absolute residual score, it allows to see regions with smaller intervals
+than others which are interpreted as regions with more certainty than others.
+It is important to note that, this conformity score is inversely proportional to the
+order of magnitude of the predictions. Therefore, the uncertainty is proportional to
+the order of magitude of the predictions, implying that this core should be used
+in use cases where we want greater uncertainty when the prediction is high.
+
+3. The conformal residual fitting score
+=======================================
+
+The conformal residual fitting score (:class:`mapie.conformity_scores.ConformalizedResidualFittingScore`)
+(CRF) is slightly more complex than the previous scores.
+The normalization of the residual is now done by the predictions of an additional model
+:math:`\sigma` which learns to predict the base model residuals from :math:`X`.
+:math:`\sigma` is trained on :math:`(X, |Y-\hat{\mu}(X)|)` and the formula of the score is :
+
+.. math:: \frac{|Y-\hat{\mu}(X)|}{\hat{\sigma}(X)}
+
+This score provides adaptive intervals : intervals of different sizes in each point
+with the following formula :
+
+.. math:: [\hat{\mu}(X) - q(s) * \hat{\sigma}(X), \hat{\mu}(X) + q(s) * \hat{\sigma}(X)]
+
+Where :math:`q(s)` is the :math:`(1-\alpha)` quantile of the conformity scores.
+(see :doc:`theoretical_description_regression` for more details).
+
+This score is by default symmetric (*see definition above*). Unlike the scores above,
+and due to the additionnal model required this score can only be used with split methods.
+
+Normalisation by the learned residuals from :math:`X` adds to the score a knowledge of
+:math:`X` and its similarity to the other examples in the dataset.
+Compared to the gamma score, the other adaptive score implemented in MAPIE,
+it is not proportional to the uncertainty.
+
+
+Key takeaways
+=============
+
+- The absolute residual score is the basic conformity score and gives constant intervals.
```

Comment:
Maybe it's worth mentioning that's in the basic one that is used throughout MAPIE as default?

Reply:
Done